doc/nrf/releases_and_maturity/migration/migration_guide_3.2.rst

```diff
@@ -70,6 +72,24 @@ This section describes the changes related to libraries.
 
 .. _migration_3.2_recommended:
 
+nRF54H20
```

Comment:
This section could be moved above the Samples and application. We normally follow the same structure as the ToC of NCS docs. For example, https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/migration/migration_guide_3.0.html.